### PR TITLE
chore: add resource requests/limits to harbor

### PR DESCRIPTION
**Pull request by paddatrapper**
Fixes: #770

### CODE REVIEW

**Review by moshloop — CHANGES_REQUESTED** [review]
Can we set minimum requests to 100m/200m cpu and 64/128Mi mem ?
Then when increasing them go in intervals of  

CPU: 100,200,500,1000,1500,2000
Mem: 64,128,256,512,1Gb,2gb,4gb etc..

Clair I think will need to burst to 1000m
Core and Registry cpu should be 200/500 and mem, 256/512
Registryctl can remain low 


e.g. 
clair

File: manifests/harbor/exporter.yaml [hunk 1]
```diff
@@ -77,11 +77,11 @@ spec:
             readOnlyRootFilesystem: true
           resources:
             limits:
-              cpu: 400m
+              cpu: 500m
```
Comment (moshloop) [inline manifests/harbor/exporter.yaml:80]:
```suggestion
              cpu: 200m
```

**Review by moshloop — CHANGES_REQUESTED** [review]
please check failing tests

**Review by moshloop — CHANGES_REQUESTED** [review]
I think we might need to reduce the requests for the test run in https://github.com/flanksource/karina/blob/master/test/patch1.yaml as we are limited to 7GB of memory total

**Review by moshloop — CHANGES_REQUESTED** [review]
So the requests/limits should have remained in manifests/harbor and then those new default requests should have been overridden in patch1, bring down the requirements for tests

**Comment by moshloop** [comment]
@paddatrapper  can you check the failing harbor test ?

**Comment by paddatrapper** [comment]
Harbor is now passing

**Pull request merged**